packages/date-time/src/components/DatePicker/DatePicker.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { IDatePickerStyleProps, IDatePickerStyles } from './DatePicker.types';
+import { IStyle, normalize, getGlobalClassNames, FontSizes } from '@uifabric/styling';
+
+const GlobalClassNames = {
+  root: 'ms-DatePicker',
+  callout: 'ms-DatePicker-callout',
+  withLabel: 'ms-DatePicker-event--with-label',
+  withoutLabel: 'ms-DatePicker-event--without-label',
+  disabled: 'msDatePickerDisabled '
+};
+
+export const styles = (props: IDatePickerStyleProps): IDatePickerStyles => {
+  const { className, theme, disabled, label, isDatePickerShown } = props;
+  const { palette, semanticColors } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const DatePickerEvent: IStyle = {
+    color: palette.neutralSecondary,
+    fontSize: FontSizes.icon,
+    lineHeight: '18px',
+    pointerEvents: 'none',
+    position: 'absolute',
+    right: '9px'
+  };
+
+  return {
+    root: [classNames.root, theme.fonts.medium, isDatePickerShown && 'is-open', normalize, className],
+    textField: [
+      {
+        position: 'relative',
+        selectors: {
+          '& input[readonly]': {
+            cursor: 'pointer'
+          },
+          input: {
+            selectors: {
+              '::-ms-clear': {
+                display: 'none'
+              }
+            }
+          }
+        }
+      },
+      disabled && {
+        selectors: {
+          '& input[readonly]': {
+            cursor: 'default'
+          }
+        }
+      }
+    ],
+    callout: [classNames.callout],
+    icon: [
+      DatePickerEvent,
+      !label && [classNames.withoutLabel, { top: '7px' }],
+      label && [classNames.withLabel, { bottom: '5px' }],
+      !disabled && [
+        classNames.disabled,
```

Comment:
This looks slightly suspicious, a "disabled" class name in a !disabled condition.

Reply:
I'll take note of these and mark them as items todo a little later- this is a direct copy-paste of the existing DatePicker control in office-ui-fabric-react. :)